README.md

```diff
@@ -12,63 +12,63 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
-| `flushMetricsToLogs`    | Send custom metrics using CloudWatch logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set `apiKey` (or `apiKMSKey`).                                                                                                                                 |
-| `site`                  | Set which Datadog site to send data. This is only used when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.                                                                            |
-| `apiKey`                | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8].                                                                                                                                         |
-| `apiKMSKey`             | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption.                                                                                                                                                 |
-| `enableDDTracing`       | Enable Datadog tracing on your Lambda functions. Defaults to `true`.                                                                                                                                                                                                                                                                   |
-| `injectLogContext`      | When set, the lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true`.                                                                                                                                                                                                                        |
+*Note*: The Descriptions may use the NPM package parameters however they also apply to the PyPi package parameters. 
+
+| NPM package Parameter        | PyPi package Parameter          | Description                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
This table is great!

Reply:
One minor nit:
`NPM package Parameter` -> `npm package parameter`
`PyPi package Parameter` -> `'PyPI package parameter`